lib/css/components/_stacks-code-blocks.less

```diff
@@ -0,0 +1,102 @@
+pre[class*="language-"],
```

Comment:
These selectors need to be cleaned up. They're a little gnarly, and probably not how I'd do it from scratch.

Reply:
Technically speaking, this doesn't even cover all the bases (in practice). In prod we use `lang-*` classes instead... That being said, the post contents bake all the code blocks as bare `pre>code` and we add in the highlighting classes client-side, so I have absolutely no problem with this approach.

In addition, the [html spec](https://html.spec.whatwg.org/#the-code-element) states:

> There is no formal way to indicate the language of computer code being marked up. Authors who wish to mark code elements with the language used, e.g. so that syntax highlighting scripts can use the right rules, can use the class attribute, e.g. by adding a class prefixed with "language-" to the element.

So, we're adhering to best practices there too.

Reply:
Ok, so the `pre[class*="language-"]` selector can be removed entirely?

Reply:
I'm 100% down with that. I feel that explicitly scoping these to `s-code-block` is preferred. We can remove the `code[...]` selector too, in favor of my added suggestion.